lib/macho/macho_file.rb

```diff
@@ -556,7 +556,7 @@ def low_fileoff
 
       segments.each do |seg|
         sections(seg).each do |sect|
-          next if sect.size == 0
+          next if sect.size.zero?
```

Comment:
Was this actually pointed out by RuboCop? It feels like this change reduces readability without any clear benefit. And the actual fix should most likely be `sect.empty?`.